hw5_Zadacha2.py

```diff
@@ -0,0 +1,43 @@
+# Создайте декоратор, который хранит результаты
+# вызовы функции
+# (за все время вызовов, не только текущий запуск программы)
+
+import os
+
+
+def dec(a):
+    def wrapper():
+        if os.path.exists("D:/rez.txt"):
+            f = open('D:/rez.txt', "r")
+            count = int(f.read())
+            count += 1
+            print(count)
+            f.close()
+            save = open('D:/rez.txt', "w")

```

Comment:
you can just append
you can use context manager

Reply:
Didn't want to use context manager at that time.

Reply:
why?